docker-sync.yml

```diff
@@ -0,0 +1,22 @@
+version: '2'
+
+# Documentation:
+# https://docker-sync.readthedocs.io/en/latest/getting-started/configuration.html
+
+options:
+  compose-file-path: ${COMPOSE_FILE}
+  verbose: true
+  cli_mode: 'auto'
+  project_root: 'pwd'
+
+syncs:
+  app-magento2-sync:
+    src: ${PROJECT_LOCATION}
+    host_disk_mount_mode: 'cached'
+
+    sync_userid: 1000
+    sync_excludes: ['.git']
```

Comment:
I think you can also add the `.idea` folder

Reply:
I wanted to avoid adding such specific things, but why not. :wink: